base/set.jl

```diff
@@ -261,9 +261,11 @@ function symdiff!(s::AbstractSet, itr)
     s
 end
 
-==(l::Set, r::Set) = (length(l) == length(r)) && (l <= r)
-<( l::Set, r::Set) = (length(l) < length(r)) && (l <= r)
-<=(l::Set, r::Set) = issubset(l, r)
+# convenience functions for AbstractSet
+# (if needed, only their synonyms issetequal, ⊊ and ⊆ must be specialized)
+==(l::AbstractSet, r::AbstractSet) = issetequal(l, r)
+<( l::AbstractSet, r::AbstractSet) = l ⊊ r
+<=(l::AbstractSet, r::AbstractSet) = l ⊆ r
```

Comment:
I don't think the definition is correct. I think it's instead: for sets `A` and `B`, `A < B` if `∀ a . A, ∀ b . B, a < b`

Reply:
If I'm not mistaken, I changed nothing to the definition of `<=` for sets. Do you suggest we should open a separate issue to change the meaning of (or delete) these methods?